kuka_kr10_support/launch/load_kr10r1100sixx.launch

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <launch>
-  <param name="robot_description" command="$(find xacro)/xacro.py '$(find kuka_kr10_support)/urdf/kr10r1100sixx.xacro'"/>
+  <param name="robot_description" command="$(find xacro)/xacro '$(find kuka_kr10_support)/urdf/kr10r1100sixx.xacro'"/>
```

Comment:
So if we want to be 'correct', we would not include this change in this PR. It's off-topic.